discovery/spec.md

```diff
@@ -588,10 +588,17 @@ The result MUST be a JSON object of the following form:
 ```
 {
   "servicefilterattributes": [ "name", ... ],
-  "pagination": "true"
+  "pagination": true
 }
 ```
 
+The following additional constraints apply:
+- The list of feature names are case sensitive.
+- The `servicefilterattribute` MUST be present, and MUST have at least the
```

Comment:
```suggestion
- The `servicefilterattributes` property MUST be present, and MUST have at least the
```

Reply:
fix

Reply:
I think this is different. This is talking about filter attribute names, but the previous one is talking about feature names. But I did reword the sentence per your previous comment.